core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -419,6 +411,16 @@ private LocalQueryRunner(
 
         accessControl.setConnectorAccessControlProvider(createAccessControlProvider(connectorManager));
 
+        globalFunctionCatalog.addFunctions(new InternalFunctionBundle(
+                new JsonExistsFunction(functionManager, metadata, typeManager),
+                new JsonValueFunction(functionManager, metadata, typeManager),
+                new JsonQueryFunction(functionManager, metadata, typeManager)));
```

Comment:
Why is this here?

Reply:
The json functions use the type system internally so they have to be added after the fact.